### ISSUE TITLE
Alternate message problem with backslash

### ISSUE THREAD

**Issue by pabx06** [ghiscoding/angular-validation#99]
```
<html><head>
<link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css" />
<link rel="stylesheet" href="bower_components/foundation/css/foundation.min.css" />
</head><body>

<form id="signup-form" name="myForm" >
<!-- credit card EXP -->
          <div class="form-group  has-feedback" ng-class="{'has-error': myForm.ccdate.$invalid, 'has-success': !myForm.ccdate.$invalid}">
            <label for="ccdate">Date d'expiration de la carte, exemple: {{date | date:"MM/yyyy"}}</label>
            <input id="ccdate" type="text" class="form-control" name="ccdate" ng-model="formData.ccdate" validation="pattern=/^(0[1-9]|1[0-2])\/(20[0-9]{2})$/:alt=Doit etre de la forme MM/YYYY ou MM le mois sur chifres et YYYY année sur 4 chifres, exemple 12/2016" validation-error-to="myErrorId220">
            <span class="glyphicon form-control-feedback" aria-hidden="true" ng-class="{'glyphicon-remove': myForm.ccdate.$invalid, 'glyphicon-ok': !myForm.ccdate.$invalid}"></span>
            <small id="myErrorId220" class="error" style="padding: inherit"></small>
          </div>
</form>
```

The pathern is : https://regex101.com/r/gN5wH2/26

/^(0[1-9]|1[0-2])\/(20[0-9]{2})$/

should match : 
02/2013
02/2013
and should not match:
13/2030
02/1010
02/0010

but it does. 
Am i doing something wrong ?

i started a jsfidle that i cant setup 100% OK :https://jsfiddle.net/jm1f9ee3/5/#&togetherjs=yw4SUscv8h

**Comment by ghiscoding** [comment]
Your jsFiddle doesn't seem to work and I don't see how the libraries are getting loaded in that jsFiddle. I don't even think Angular-Validation was ever loaded. I get so many errors in the console when running it.

You should to reuse one of my Plunker examples and just recode the portion you want.

There is something else that can tell you if the validation get called or not, just look at the input border if it becomes green or red then you know it got called. If it stays grey (like in your jsFiddle) then there's obviously some problems somewhere in your code.

**Comment by pabx06** [comment]
i found out where the error came from:
this doesnt work:
```
validation="pattern=/^(0[1-9]|1[0-2])\/(20[0-9]{2})$/:alt={{'Doit etre de la forme MM/YYYY ou MM le mois sur chifres et YYYY année sur 4 chifres, exemple 12/2016'}}|required"
````
this work: (i removed the slashs  chars (/)  from the alt)
```
validation="pattern=/^(0[1-9]|1[0-2])\/(20[0-9]{2})$/:alt={{'Doit etre de la forme MMYYYY ou MM le mois sur chifres et YYYY année sur 4 chifres, exemple 122016'}}|required"
```

**Comment by ghiscoding** [comment]
Oh you might have to escape it, try like this `MM\/YYYY` and see if that work.

**Comment by pabx06** [comment]
ok i escaped:
```
validation="pattern=/^(0[1-9]|1[0-2])\/(20[0-9]{2})$/:alt={{'Doit être de la forme MM\/YYYY ou MM le mois en 2chifres et YYYY année sur 4 chiffres, exemple 12\/2016'}}|required"

```
didnt work

**Comment by ghiscoding** [comment]
Please rename the issue to better words since you found the real issue being the `alt` message. Something like `Alternate message problem with backslash`

I will eventually look at it, I'm still in holiday mode :P

**Comment by ghiscoding** [comment]
This is now fixed in latest revision [v1.4.19](https://github.com/ghiscoding/angular-validation/releases/tag/v1.4.19)

Thanks for the feedback :)
